### ISSUE TITLE
nolint comments are not working

### ISSUE THREAD

**Issue by dvyukov** [golangci/golangci-lint#468]
//nolint comments are not working, I am getting:
```
pkg/ifuzz/decode.go:13:1: cyclomatic complexity 75 of func `Decode` is high (> 24) (gocyclo)
func Decode(mode int, text []byte) (int, error) { //nolint:gocyclo
```

```
golangci-lint has version v1.15.0-21-gf516771 built from f516771 on Thu Mar 28 14:06:02 CET 2019
```

**Comment by jirfag** [comment]
Hi, thank you!
The issue was because of poor `.rl` and `.y` files support. I've fixed it in https://github.com/golangci/golangci-lint/pull/504 and https://github.com/golangci/golangci-lint/pull/505.

I've run `golangci-lint` on the github.com/google/syzkaller: `gocyclo` doesn't report about `pkg/ifuzz/decode.go`.

**Comment by dvyukov** [comment]
Now everything seems to work as expected. Thanks!
